it/TransmissionRequestWorkItemRepositoryISpec.scala

```diff
@@ -90,7 +92,7 @@ class TransmissionRequestWorkItemRepositoryISpec extends UnitSpec
       stubMdgToFail()
 
       And("the request is added to the work item queue")
-      val workItem: WorkItem[TransmissionRequestEnvelope] = await(testInstance.pushNew(envelope, Instant.now.toDateTime))
+      val workItem: WorkItem[TransmissionRequestEnvelope] = await(testInstance.pushNew(envelope, new DateTime(Instant.now)))
```

Comment:
Why did we change this?

Reply:
I changed it because there were two Instant imports being used, org.joda.time.Instant and java.time.Instant. My IDE was giving me warnings as I think it was confused which one was being used where but I've run the tests again and they still seem to pass so I will change it back.